src/ROUTES.ts

```diff
@@ -53,6 +53,11 @@ const ROUTES = {
         getRoute: (query: string, reportID: string) => `search/${query}/view/${reportID}` as const,
     },
 
+    TRANSACTION_HOLD_REASON_RHP: {
+        route: '/search/:query/hold/:transactionID/:searchHash',
+        getRoute: (query: string, transactionID: string, searchHash: number) => `search/${query}/hold/${transactionID}/${searchHash}` as const,
```

Comment:
Thinking more about this, why do we need the hash in the URL? The hash is derived from the query which is already in the URL. Can we just remove the hash entirely?

Reply:
I also was thinking about this. Currently hash is generated like this: `getQueryHash(query, policyIDs, sortBy, sortOrder)` so it requires 4 params. I decided it makes more sense to add 1 hash param to this `/hold` route, than to add all 4 Search query params.

It is messy I know :/